astroquery/utils/tap/model/job.py

```diff
@@ -201,39 +304,164 @@ def wait_for_job_end(self, verbose=False):
         """
         currentResponse = None
         responseData = None
+        lphase = None
+        # execute job if not running
+        if self._phase == 'PENDING':
+            print("Job in PENDING phase, sending phase=RUN request.")
+            try:
+                self.start(verbose)
+            except:  # noqa: E722

```

Comment:
better to do `except Exception as ex:` here, then `print(ex)` or similar below

Reply:
OK